packages/bpk-component-map/src/BpkMap.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
+import { withScriptjs, withGoogleMap, GoogleMap } from 'react-google-maps';
+
+const BpkMap = withScriptjs(
+  withGoogleMap(({ mapRef, ...rest }) => <GoogleMap ref={mapRef} {...rest} />),
+);
```

Comment:
@matteo-hertel and I talked this over and feel that it provides a nicer API to consumers if `BpkMap` wraps the component in a HOC for them, but I'm highlighting it as maybe something we don't want to do. I'm trying to think of use cases where a user *doesn't* wish to do this.